### ISSUE TITLE
deprecated jekyll tag usage spamming console

### ISSUE THREAD

**Issue by tfoote** [ros-infrastructure/rosindex#307]
The following output is spamming the console

There are about 160000 instances of it in each build constituting about 30% of our build output.

`/home/jenkins-agent/.bundle/ruby/2.7.0/gems/jekyll-3.8.5/lib/jekyll/tags/include.rb:194: warning: Using the last argument as keyword parameters is deprecated`

It's both a lot of console logging space and bandwidth as well as a risk of it breaking when the deprecation cycle finishes.

**Comment by tfoote** [comment]
This may be an issue in the jekyll version we're using we need to upgrade to 3.8.7 or higher

https://github.com/jekyll/jekyll/issues/7947#issuecomment-640257352

**Comment by rkent** [comment]
I posted a patch on my fork upgrading the Dockerfile to jammy, also updating Gemfile.lock  Full run available (temporarily) at http://test3.rosdabbler.com

The main issue was newer jekyll no longer supports red carpet, so the md processor had to be changed to kramdown. That forced a minor fix concerning character encodings, but otherwise seems to work OK. "spamming the console" is greatly reduced, not only because of the jekyll upgrade, but also because I removed most of the --verbose options in the Makefile.

At this point I need to read through the issue history to see what historical issues have existed that might be affected by switching to kramdown. If there is a history here that anyone knows about, I would love to hear about it.
